bower.json

```diff
@@ -29,7 +29,9 @@
     "purescript-lists": "^0.7.0",
     "purescript-maybe": "^0.3.0",
     "purescript-strings": "~0.7.0",
-    "purescript-transformers": "^0.8.1"
+    "purescript-transformers": "^0.8.1",
+    "purescript-unicode": "git://github.com/purescript-contrib/purescript-unicode#unicode",
```

Comment:
You should publish this to bower. Never actually commit git dependencies, especially when it's not a SHA.